### PR TITLE
Tina 1.1.3 changelog

### PR DESCRIPTION
**Pull request by jamespohalloran**
Update Tina, and create changelog post

### CODE REVIEW

File: content/blog/tina-v-1.1.3.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: 'TinaCMS V1.1.3'
+date: '2023-01-11T22:00:00.000Z'
+last_edited: '2023-01-11T22:00:00.000Z'
+author: James O'Halloran
+---
+
+TinaCMS Version `1.1.3` (and `@tinacms/cli@1.0.6`) brings some improvements for various frameworks, and form list-fied improvements.
```
Comment (kldavis4) [inline content/blog/tina-v-1.1.3.md:8]: should this be `list-field`?

File: content/blog/tina-v-1.1.3.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+https://user-images.githubusercontent.com/5075484/211587949-a19681e1-f8c9-4f91-820a-f7f55eeb0e6d.mp4
+
+## TOML & YML Support
+
+### Customizable Frontmatter Format Updates
+
+Collections can now specify `{ frontmatterFormat: 'toml' | 'yaml' | 'json' }` and `{frontmatterDelimiters: string | [string, string] }`.
+
+This allows the user to have more conrtol of how their frontmatter is parsed.
```
Comment (kldavis4) [inline content/blog/tina-v-1.1.3.md:50]:
```suggestion
This allows the user to have more control of how their frontmatter is parsed.
```

File: content/blog/tina-v-1.1.3.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Editors will also be able to edit fields in the shortcodes indivudually, instead of through a global `text` field.
+
+See [here](https://tina.io/docs/reference/types/rich-text/#custom-shortcode-syntax) for more info.
+
+## Min/Max Support in list fields
+
+The field UI object now accepts a min and max when the field is `list: true`. If the list/group list/block field is at the max or min value, the add or remove buttons are disabled. If the min and max values are the same, the field is considered 'fixed length' and the add/remove buttons are removed entirely, unless the data is invalid (you can still add items if there's not enough to satisfy the validation).
+
+https://user-images.githubusercontent.com/5075484/211587949-a19681e1-f8c9-4f91-820a-f7f55eeb0e6d.mp4
+
```
Comment (scottgallant) [inline content/blog/tina-v-1.1.3.md:43]: Video isn't embedded properly (should be in a video tag)

Comment (jamespohalloran) [reply content/blog/tina-v-1.1.3.md:45]: video doesn't seem to work in our blog, so replaced it with an image

**Pull request merged**